docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -0,0 +1,106 @@
+# Deployment Instructions
+
+1. Clone the repo
+
+   ```bash
+   git clone https://github.com/nick-stroud/hpc-toolkit.git
+   cd hpc-toolkit
+   git checkout hcls_mega_blueprint
+   ```
+
+1. Build the HPC Toolkit
+
+   ```bash
+   make
+   ```
+
+1. Generate the deployment folder
+
+   The home and apps server IPs will be populated after the setup stage and are
+   needed for later stage deployments. For now we will populate with a dummy
+   value to allow HPC Toolkit to run.
+
+   ```bash
+   ./ghpc create docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml -w \
+       --vars project_id=<project> \
+       --vars homefs_server_ip=1.1.1.1 \
+       --vars appsfs_server_ip=1.1.1.1 \
+       --vars bucket_name_inputs=<input_bucket_name> \
+       --vars bucket_name_outputs=<output_bucket_name> \
+       --vars bucket_name_software=<software_bucket_name>
+   ```
+
+1. Run terraform commands for `enable_apis` group
+
+   Call the provided `terraform init` and `terraform apply` commands for the
+   `enable_apis` group. This will ensure that all of the needed apis are
+   enabled before deploying the cluster.
```

Comment:
why not provide the commands? these should be known, right? 
AKA:
```
terraform -chdir=hcls-01/enable_apis init &&  terraform -chdir=hcls-01/enable_apis apply
```

Reply:
Done.